lizard_auth_server/tests/test_views.py

```diff
@@ -40,6 +43,32 @@ def test_smoke_as_admin(self):
         self.assertEquals(result.status_code, 200)
 
 
+class ConfirmDeletionUserconsentViewTestCase(TestCase):
+
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.user = factories.UserF()
+        self.client = OIDC_Client.objects.create(client_id='id')
+        expires_at = datetime(3022, 2, 17)
```

Comment:
Is dit nog een specifieke datum uit een science fiction boek? Of is het random?

(Ik vraag het maar even omdat ik zelf vaak 1972 er als grapje infiets: https://github.com/search?l=Python&q=org%3Anens+1972&type=Code&utf8=%E2%9C%93 )